src/ray/raylet/node_manager.cc

```diff
@@ -1458,6 +1462,15 @@ void NodeManager::ProcessSubmitTaskMessage(const uint8_t *message_data) {
   SubmitTask(Task(task_message));
 }
 
+void NodeManager::HandleRequestResourceReport(
+    const rpc::RequestResourceReportRequest &request,
+    rpc::RequestResourceReportReply *reply, rpc::SendReplyCallback send_reply_callback) {
+  auto resources_data = reply->mutable_resources();
+  FillResourceReport(resources_data);
```

Comment:
Nit: Any reason to not pass the resources pointer straight in as an rvalue?

```suggestion
  FillResourceReport(reply->mutable_resources());
```